### PR TITLE
[DOWNSTREAM PORT] Hardsuit Helmet Fixes & Updates

### PR DESCRIPTION
**Pull request by KillianKirilenko**
## ABOUT
Ports some miscellaneous fixes and updates I made for CitRP, namely fixing a small typo that prevented vulp and vulp-derived customs from receiving the correct hardsuit helmet sprites.

Includes minor updates to the industrial/mining, medical, eva, and sci/ami suit+helmet sprites for the same. Helmet colors and designs should blend better (particularly the industrial/mining) and the sci/ami suit actually has a (brand new, quickly-made) helmet now.

Doesn't port the changes to the hazard hardsuit or the HoS' prototype suit as I'm still not satisfied with the former and the latter is much more extensive + requires mapping changes. If someone else wants to port these they are more than welcome to.

Example of the industrial/mining resprite:
image

## CHANGELOG
:cl:
fix: fixed vulp+vulpcustoms not getting proper hardsuit helmets
tweak: recolored and redesigned a few hardsuit helmet sprites to better match their suits
add: added a sci/ami helmet based on the eva helmet (since the ami suit is just a modded eva suit)
/:cl:

### CODE REVIEW

File: code/modules/clothing/spacesuits/rig/rig_pieces_vr.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@
 		SPECIES_SERGAL			= 'icons/mob/species/sergal/helmet_vr.dmi',
 		SPECIES_ZORREN_FLAT		= 'icons/mob/species/fennec/helmet_vr.dmi',
 		SPECIES_ZORREN_HIGH 	= 'icons/mob/species/fox/helmet_vr.dmi',
-		SPECIES_VULPKANI 		= 'icons/mob/species/vulpkanin/helmet.dmi',
+		SPECIES_VULPKANIN 		= 'icons/mob/species/vulpkanin/helmet.dmi',
```
Comment (TheFurryFeline) [inline code/modules/clothing/spacesuits/rig/rig_pieces_vr.dm]: 💢 Was it really that badly overlooked for this define here??? Good grief!

Comment (KillianKirilenko) [reply code/modules/clothing/spacesuits/rig/rig_pieces_vr.dm]: Yep! I fixed this on Cit back in July last year.

**Comment by Novacat** [comment]
[quote]
latter is much more extensive + requires mapping changes.
[/quote]

Our HoS voidsuit is no longer mapped in. We moved it over to code where it should have been and nerfed it to have the same stats as a regular security voidsuit.

**Comment by KillianKirilenko** [comment]
> Our HoS voidsuit no longer has its stats mapped in. We moved it over to code where it should have been and nerfed it to have the same stats as a regular security voidsuit.

not so much the stats part as the looks - I actually wound up making a bunch of variants for it;
image

**Pull request merged**